kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/KyuubiStatement.java

```diff
@@ -543,6 +543,18 @@ public int getMaxRows() throws SQLException {
     return maxRows;
   }
 
+  @Override
+  public boolean getMoreResults(int current) throws SQLException {
+    // getMoreResults() javadoc says, that it should implicitly close
+    // any current ResultSet object, so here in case of Statement.CLOSE_CURRENT_RESULT
+    // we can implement the same logic
+    // see also https://issues.apache.org/jira/browse/HIVE-7680
```

Comment:
it is a bug fix, and is worth making this change in a separate PR.

Reply:
PR: https://github.com/apache/kyuubi/pull/6115

Reply:
BTW, which errors will be happen without this change?

Reply:
Oh, I see, ResultSetWrapper calls `statement.getMoreResults(Statement.CLOSE_CURRENT_RESULT)`